src/test/java/api/InstancesApiExamples.java

```diff
@@ -81,7 +82,8 @@ public void canCreateAnInstance()
         .put("name", "Chambers, Becky")))
       .put("source", "Local")
       .put("instanceTypeId", ApiTestSuite.getTextInstanceType())
-      .put(TAGS_KEY, new JsonObject().put(TAG_LIST_KEY, new JsonArray().add(tagNameOne)));
+      .put(TAGS_KEY, new JsonObject().put(TAG_LIST_KEY, new JsonArray().add(tagNameOne)))
+      .put("natureOfContentTermIds", new JsonArray(asList("96879b60-098b-453b-bf9a-c47866f1ab2a", "f5908d05-b16a-49cf-b192-96d55a94a0d1")));
```

Comment:
I imagine these IDs do not refer to nature of content term records, meaning that if these tests are used with a real storage module they will likely fail.

I believe the other other tests deliberately use a fake module and create the records they need. Could this be done for these tests as well?

Reply:
Done

Reply:
Thanks